File: PY/2344.py
# BEE 2344 - Notas da Prova - Python
def nota(notas: int):
    if notas >= 1 and notas <= 35:
        return 'D'
    elif notas >= 36 and notas <= 60:
        return 'C'
    elif notas >= 61 and notas <= 85:
        return 'B'
    elif notas >= 86 and notas <= 100:
        return 'A'
    return 'E'


print(nota(int(input())))

File: C/2344.c
#include <stdio.h>

char notas(int nota)
{
    if (nota == 0)
    {
        return 'E';
    }
    else if (nota >= 1 && nota <= 35)
    {
        return 'D';
    }
    else if (nota > 35 && nota <= 60)
    {
        return 'C';
    }
    else if (nota > 60 && nota <= 85)
    {
        return 'B';
    }
    else
    {
        return 'A';
    }
}

int main()
{
    int nota;
    scanf("%d", &nota);
    printf("%c\n", notas(nota));
    return 0;
}